### ISSUE TITLE
AttributeError: 'Context' object has no attribute 'process'

### ISSUE THREAD

**Issue by kracon7** [pycampers/oscilloscope#3]
I was trying the example from the homepage.

```
import random
from time import sleep
from oscilloscope import Osc
osc = Osc()

@osc.signal
def simple_random_signal(state):
    while True:
        state.draw(random.random())
        sleep(0.1)

osc.start()
```

Here's what I got:
```
(mujoco) jc@jc-XPS-15:~/mujoco_ws/test_xmls$ python osc_test.py 
Traceback (most recent call last):
  File "osc_test.py", line 9, in <module>
    @osc.signal
  File "/home/jc/virtualenvs/mujoco/lib/python3.6/site-packages/oscilloscope.py", line 198, in signal
    return zproc_ctx.process(_signal_process, **process_kwargs)
AttributeError: 'Context' object has no attribute 'process'
[ZProc] Cleaning up 'python' (8690)...


```

**Comment by devxpy** [comment]
Thanks for the report!

Looks like this is a problem due to an outdated version of zproc. Let me issue a fix...

**Comment by devxpy** [comment]
@kracon7 
Fixed in `v0.0.4`.
